### ISSUE TITLE
Evaluate and rethink the rchain.coop site

### ISSUE THREAD

**Issue by kitblake** [rchain/marketing#21]
tl;dr: We need to evaluate and rethink what we want to achieve with the main website. The content is improving but the presentation is lackluster and needs an overarching concept.

A lot of people worked hard to get the new rchain.coop site up and running before RCon3. Kudos to all involved. We needed to get it made and we did.

Now that it's up, I'm thinking – and hearing – that it falls short. The site is not exciting. It looks decent but that's not the quality level we're shooting for. Technically, the home page is extraordinarily heavy and takes a really long time to load. The other pages are primarily white and Girvin is not happy with the plum purple color. 

The animations are large and I'm not sure what value they add. I've tried to discern meaning in them but it escapes me. They remind me of decorative patterns on Arabic architecture. I've also been told that the continuous flickering makes the content hard to read.

I know that @KWattage and @pmoorman spent a ton of time developing the content focus and messaging. I wonder if we can use that thinking to develop a design concept, one that can underpin a dynamic presentation with the scope to steadily expand the site.

**Comment by pmoorman** [comment]
@kitblake couldn't agree more, and I think almost everyone would agree here.

@KWattage and I decided 2 months ago to split the work into a "design" and a "content" track, which I think we should maybe maintain, and possibly expand with a "technical" track (e.g. page load speeds, etc.).

From my perspective, the main thing that's blocking me is from working on this more actively is that it's unclear who's to lead the efforts. I think @KWattage is still in charge, but haven't been able to get a hold of him since the 28th of August, which is almost a month already. (Btw, it might be that he's working on things that I'm just not aware of).



I would like to take more ownership over both the content and the design track, provided we can clear up the process for releases, as that's blocked me from pushing changes in the past. 

@kitblake I'll write you a DM in Discord to see if we can find a moment to call, and discuss these matters on a call, which is probably much quicker. @KWattage if you could join, that would be even better, of course.

_(finally: I know that @ddayan has been passionate but frustrated about the website work for a long time, too. Can someone add him here?)_

**Comment by KWattage** [comment]
Monday or Tuesday evening pacific time works for me and should hopefully
work for the European crew too.

Couple thoughts-

I think what would be beneficial for the community is a working plan of
record. Due to our nature, we have conversations going on in many, many
forums, and no source of truth to refer to for a current and consolidated
view. I will take responsibility for building this, and you are all welcome
to provide inputs and take accountability for key areas.

I don’t like the assertion that we didn’t ask the community. Perhaps, we
didn’t ask everyone in the community. In fact, the narrative was created
iteratively, starting from the marketing plan, a product truths document,
and interviews with more than 20 startups who are building on RChain or
applied for funding from reflective.

The primary objective of the site today is to provide a differentiating
story to prospective developers who are willing and able to learn/write
purposeful apps utilizing blockchain.

Helpful feedback about site performance has been awesome. Feedback that we
need better navigation and improved member signup flow, backed by examples
of prospective developers who have taken the time to write to us, is very
helpful. “I don’t like X and I prefer to say it this way instead” is less
helpful.

I very much like the idea of forming working groups. I think forming a
content working group, and creating a community content induction and
quality control process, is valuable and high priority. I’ve been
advocating for the community to have a bigger role in storytelling,
however, our website “core content” is not a free for all canvas. We are
going to have amazing community members who have diverging opinions. More
involvement from you all, to create and then align to a written, published
plan, would be awesome. I certainly don’t want to be a blocker to progress.
I merely want to make sure we tell a clear, purposeful story and create
room for all of your creative energies.

**Comment by pmoorman** [comment]
@JoshOrndorff @KWattage Monday night would work excellently for me. I'm 2 hours ahead of @kitblake so for me something like **Monday 9pm or 10pm Pacific** would work well, but if Kit wants to join we might want to push it a little later if that still works for everyone in the US.

**Shall we do 10pm Monday night?**

@JoshOrndorff if you send me your email I'll send you a calendar invite, and @kitblake let me know if you'd want to join in, too.

I can provide a Zoom room.

**Comment by pmoorman** [comment]
@KWattage can you confirm attendance for the meeting tomorrow / tonight?  (Monday 10pm Pacific)

From Joshy & Dror I have confirmation.

**Comment by KWattage** [comment]
Cool, yes I just rsvp’d. Thanks for setting this up!

**Comment by PatrickM727** [comment]
How did this meeting go? 

In general I think a good way to treat this mechanism of suggestion(github issue tracker) is to make each issue one suggestion, make it actionable, and explain the reasoning within the issue.

p.s interested in content work group scoping. lmk

**Comment by JoshOrndorff** [comment]
@PatrickM727 We did have the meeting

@KWattage suggested we classify issues into three categories content, design, and hosting/technical. We all agreed and I created the label in the issue queue. I think this categorizing works nicely with your plan to make issues represent clear actionable suggestions. And also with the workflow you suggested earlier.

We also briefly discussed the rholang tutorial and #3 and Pieter made some suggestions which I've outlined https://github.com/rchain/marketing/issues/3#issuecomment-424412396.

**Comment by kitblake** [comment]
I applaud the move to divide issues into content, design, and hosting/technical. In a well implemented web infrastructure these categories are isolated and managed separately. I.e. @KWattage and @pmoorman created content. That content can be housed in different presentations. Right now we're serving with react.js, but the same content and design can be hosted on many web platforms.

On a code level, it should be possible to create content or tweak the design simultaneously, without content authors and frontend devs getting in each other's way.

For big changes we should have a dev/production server setup (eventually dev/test/production instances) where renovations – in content and/or design – can be refined and then migrated to the public facing server.
